pkg/blockbuilder/scheduler/status_preview_test.go

```diff
@@ -0,0 +1,73 @@
+//go:build preview
+// +build preview
+
+package scheduler
+
+import (
+	"fmt"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/twmb/franz-go/pkg/kadm"
+
+	"github.com/grafana/loki/v3/pkg/blockbuilder/types"
+)
+
+// TestPreview is a utility test that runs a local server with the status page.
+// Run it with: go test -tags=preview -v -run TestPreview
+func TestPreview(t *testing.T) {
```

Comment:
nit: i guess we could just remove `status_test.go` since this is a better way to preview the page